src/Logging/ServiceCollectionExtensions.cs

```diff
@@ -47,7 +48,8 @@ public static ILoggingBuilder AddOmexLogging(this ILoggingBuilder builder)
 		[Obsolete("OmexLogger and OmexLogEventSource are obsolete and pending for removal by 1 July 2024. Please consider using a different Logger.", DiagnosticId = "OMEX188")]
 		public static IServiceCollection AddOmexLogging(this IServiceCollection serviceCollection)
 		{
-			serviceCollection.AddLogging();
+			serviceCollection.AddLogging(builder =>
```

Comment:
Very crucial to contruct LoggerProviderConfigurationFactory in order to have InitializationLogger read the appsettings.json

Reply:
InitializationLogger already adds configuration by calling the ILoggerBuilder.AddOmexLogging method above this one.

Reply:
Yes, however, 
1. not everyone is using InitilizationLogger to have this loggerbuilder.AddConfiguration());
There maybe some consumers just .AddOmexLogger without calling BuildServiceFabricService()-> to construct InitilizationLogger -> to have loggerbuilder.AddConfiguration());
2. Currently, OmexLoggerProvider does not need to read IOption/IConfiguration. But from this PR, OmexLoggerProvider  must read IOption/IConfiguration. And the only way to have OmexLoggerProvider is to access IConfiguration is through builder.AddConfiguration());
3. There is no harm in calling this twice.